app/src/main/java/com/quran/labs/androidquran/SearchActivity.java

```diff
@@ -363,7 +363,7 @@ public View getView(int position, View convertView, ViewGroup parent) {
 
       SearchElement v = mElements.get(position);
       String text = v.text;
-      String suraName = QuranInfo.getSuraName(mContext, v.sura, false);
+      String suraName = QuranInfo.getSuraName(mContext, v.sura, false, false);
```

Comment:
since all of these calls with false except for one, how do you feel about leaving `getSuraName` with three parameters that calls `getSuraName` and passes false for the translation parameter?